### ISSUE TITLE
Placeholder - Personas

### ISSUE THREAD

**Issue by cyberfox** [oobabooga/text-generation-webui#6478]
**Description**

tl;dr - Implement personas; easily changing multiple areas of TGW to reflect the task or need the user has at the moment.

As a user of TGW, when I sit down to interact, I am usually in one of a few different personas. For example:
* Looking for life-coach guidance.
* Asking for coding assistance and quick tool generation.
* Looking for assistance building out infrastructure-as-code solutions.
* Have a fun roleplaying chat with an AI friend.
* Fleshing out story ideas with character names, places, and initial descriptions.
...etc.

Each of these personas (all myself!) are distinct, and require different user descriptions, 'character' choices, and even system prompts! In some cases they may call for different models, but that's a stretch goal. They will _DEFINITELY_ require different generation parameters. (_For example, using 'conversational' settings like the DRY filter will absolutely destroy the coding assistance, because code often has repeated blocks that are considered redundant by the DRY processor._)

I envision a drop-down (_similar to the character drop-down_) which allows you to select a complete persona for this session, updating all the things with one selection. It should be possible to download a complete persona as a .zip file, including the character information, and move it to another system, but again...stretch goals.

Areas (_at first blush_) that need to be loaded/saved in a persona:
* Generation parameters
* Character Card
* User Card
* System prompt

**Additional Context**

This is a placeholder feature request, mostly because I intend to build some version of this, and am not asking for it to be built for me. *Any help would be gratefully appreciated, of course.*

I hope it's okay for me to use this space as a stand-in; I would like to track the feature on TGW, even if it ultimately ends up being mostly an extension. Initially my intention is to make it an extension, but it may require deeper integration touches to the whole of TGW to make it work.

**Comment by oobabooga** [comment]
This feature has been implemented in PR #7367. You can now use the user profile dropdown to save and load different personas.
